components/service/pocket/src/main/java/mozilla/components/service/pocket/stories/PocketStoriesUseCases.kt

```diff
@@ -10,101 +10,67 @@ import mozilla.components.concept.fetch.Client
 import mozilla.components.service.pocket.PocketRecommendedStory
 import mozilla.components.service.pocket.api.PocketEndpoint
 import mozilla.components.service.pocket.api.PocketResponse
-import mozilla.components.support.base.log.Log
+import mozilla.components.service.pocket.logger
 
 /**
  * Possible actions regarding the list of recommended stories.
  */
-class PocketStoriesUseCases {
+object PocketStoriesUseCases {
```

Comment:
What's the rationale for turning this into an object/singleton? Potentially downsides I see:
* Makes testing a bit awkward (e.g. passing in custom mocks/stubs)
* Makes extending potentially awkward (if you add use cases that will need dependencies when constructing)

Reply:
Just because the previous code (which I also wrote) looks a bit ugly and is more complex than it should be.
Eg: `getPocketStories()` could very well be a simple top level function. Kept it in an object just to group these methods together and make them easy to reason with and their intent clearer when being called in code - `PocketStoriesUseCases.getPocketStories`

Agree there are drawbacks (I've removed some tests (that mockk could've helped with)), can restore the previous approach which would easily support extending the functionality in the future.

Reply:
Restored.